### DISCUSSION TITLE
What is actualy "Center"?

### DISCUSSION THREAD

**Discussion by DoughyInTheMiddle** [jneilliii/OctoPrint-BedLevelVisualizer#426] [Q&A]
I have an Ender 3 Pro, and so the bed is 235x235. That makes "center" 117.5 on both axes. When I run BLV, the X and Y axes both only show whole numbers. On my 9x9 mesh that I'm currently running, "center" is 118 on both.

Is it merely rounding for the display -- and actually probing X117.50/Y117.50 -- or does the G29 actually round to the nearest whole number position, so it's actually probing X118.00/Y118.00?

I'm trying to verify points not on the mesh (specifically corner points for tramming) and make sure my BLTouch is also working properly. There's not a ton of change between 118 and 117.5, but I still would like to make sure I'm at the correct point.

**Comment by jneilliii** [comment]
The points are approximated based on bed size. The probing of firmware will still probe on the exact points you think they are probing, but the returned mesh report doesn't typically include the x and y positions of those points, and therefore the plugin has no way of knowing where it was literally probed at.
